plugins/edge-config/index.ts

```diff
@@ -10,38 +10,82 @@
 
 (function (events: Event[]): ValidationPluginResult {
   const { toolkit: kit } = window.griffon;
-  const { configuration } = kit['aep-mobile'];
+  const { sharedStateConfig } = kit['aep-mobile'];
   const configEvents = kit.match(
-    configuration.matcher,
+    sharedStateConfig.matcher,
     events
-  ) as Configuration[];
+  ) as SharedStateConfig[];
 
-  const hasAllConfig = (event: Configuration) =>
-    configuration.getEventDataKey('"edge.configId"', event);
+  const hasEdgeConfigId = window.griffon.toolkit.search('payload.metadata."state.data"."edge.configId"' , configEvents[0]);
+
+  const hasEdgeConfigDomain = window.griffon.toolkit.search('payload.metadata."state.data"."edge.domain"' , configEvents[0]);
+
+  const regex = /^(?:[a-z0-9](?:[a-z0-9-]{0,61}[a-z0-9])?\.)+[a-z0-9][a-z0-9-]{0,61}[a-z0-9]$/gi;
+
+  var validEdgeConfigDomain = false;
+
+  //If edge domain is existed, check if it is in the right format
+  if (hasEdgeConfigDomain) {
+    if (regex.test(hasEdgeConfigDomain) && !hasEdgeConfigDomain.includes("http" || "https")) {
+    validEdgeConfigDomain = true;
+    } else {
+      validEdgeConfigDomain = false;
+    }
+  }
 
   return !configEvents.length
-    ? {
-        events: [],
-        message:
-          "No configuration info could be found. Either Griffon isn't registered or it did not pass in cached events upon activating.",
-        result: 'not matched'
-      }
-    : configEvents.some(hasAllConfig)
-    ? {
-        events: [],
-        message: 'Edge extension was configured correctly',
-        result: 'matched'
-      }
-    : {
-        message:
-          'Did not detect the required configuration values. You may need to install the extension in launch and publish those settings.',
-        events: configEvents.map((event) => event.uuid),
-        result: 'not matched'
-      };
-});
+  ? {
+      events: [],
+      message:
+        "No configuration info could be found. Either Assurance isn't registered or it did not pass in cached events upon activating.",
+      result: 'not matched'
+    }
+  : hasEdgeConfigId && !hasEdgeConfigDomain
+  ? {
+      events: [],
+      message: 'Did not detect the required Edge Network domain configuration value. It should be configrued in the Data Collection UI.',
+      result: 'not matched',
+      links: [
+        {
+          type: 'doc',
+          url: 'https://developer.adobe.com/client-sdks/documentation/edge-network/#domain-configuration'
+        }
+      ],
+    }
+  : hasEdgeConfigId && validEdgeConfigDomain
+  ? {
+      events: [],
+      message: 'Edge extension was configured correctly',
+      result: 'matched'
+    }
+  : hasEdgeConfigId && !validEdgeConfigDomain
+  ? {
+      events: [],
+      message: 'Edge extension is detected, but Edge Network domain value is not configured properly. The domain name should in the right format and just the domain without any protocol or trailing slash.',
```

Comment:
nit: we can use a simple example similar to the Data Collection validation error:
The domain name should be a valid domain with no protocol or slashes. For example company.data.adobedc.net would be valid where as https://company.data.adobedc.net/ would not.